osf/migrations/0041_add_quickfiles.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11 on 2017-06-13 17:35
+from __future__ import unicode_literals
+
+import logging
+
+from django.db import migrations, models
+from django.core.management.sql import emit_post_migrate_signal
+
+from osf.models import OSFUser, QuickFiles
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def add_quickfiles(*args, **kwargs):
+    emit_post_migrate_signal(2, False, 'default')
```

Comment:
I think that is only pertinent to permissions because permissions are automatically created from `class Meta` options.